index.bs

```diff
@@ -1516,6 +1527,8 @@ When this operation is invoked, the authenticator must perform the following pro
     so, return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
 - If the |requireResidentKey| flag is set to |true| and the authenticator cannot store a [=Client-side-resident Credential
     Private Key=], return an error code equivalent to "{{ConstraintError}}" and terminate the operation.
+- If the |requireRateLimiting| flag is set to |true| and the authenticator cannot support [=Rate Limiting=], return an error code 
```

Comment:
I would not think so. For example, anyone may implement FIDO UAF (& U2F) v1 protocols and  authenticators (certified or not).  An RP may accept registrations from uncertified authnrs at their discretion. For many (most?) authnrs, RPs are able to ascertain via [the metadata service](https://fido-mds-parser.appspot.com/) whether an authnr is certified or not.  And if there is no authnr metadata, then that in itself is a piece of information the RP can (and should, if it is a high-value site) factor into its risk decision on whether to honor the authnr and accept the registration, or not. 

In UAF we did not designate "rate limiting" as an identified [authenticator characteristic](https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-registry-v1.1-id-20170202.html#authenticator-characteristics) (in the [FIDO Registry spec](https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-registry-v1.1-id-20170202.html)). Rather, we state in UAF authenticator commands "[security guidelines](https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-authnr-cmds-v1.1-id-20170202.html#security-guidelines)",  in the "Matcher" section, that:
> Authenticators must implement anti-hammering protections for their matchers.

..which is enforced as a part of authenticator Level 1 (and above) [security certification](https://fidoalliance.org/getting-started/). (Level 1 is slated to become the default entry-level authnr certification, encompassing the present "functional certification". [See here for FIDO certification program info](https://fidoalliance.org/getting-started/))